search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -81,7 +114,7 @@ function setCountryTrends(countryCode) {
 function toggleDisplay(id) {
   document.getElementById(id).classList.toggle('hidden');
 }
-
+ 
```

Comment:
remove these empty spaces here and all other places

Reply:
+1

Reply:
I deleted the empty spaces before //comments, but I kept some of the ones where in the function it seems natural to have a break (for ex when we're defining two separate long varialbes), if that's okay

Reply:
I think they meant the spaces on this line as I mentioned

Reply:
oh isee thanks!